pom.xml

```diff
@@ -611,7 +611,11 @@
         <executions>
           <execution>
             <goals>
-              <goal>javadoc</goal>
+              <!--
+                Goal needs to be the "-no-fork" variant, otherwise it causes other plugins running
+                in e.g. the "initialise" phase to execute twice (e.g. the enforcer plugin + frontend node/npm install).
+              -->
+              <goal>javadoc-no-fork</goal>
```

Comment:
I have noticed in the past that the javadoc code is not designed to be embedded as it does not release resources correctly causing memory issues for multi module projects.
caution: **Here be dragons**....